src/js/utils/mixins.js

```diff
@@ -1,7 +1,13 @@
 import { css } from 'styled-components';
 
-export const parseMetricToNum = fontAsString =>
-  parseFloat(fontAsString.replace(/[^0-9/.]/g, ''), 10);
+export const parseMetricToNum = fontAsString => {
+  if (fontAsString.match(/\s/) && process.env.NODE_ENV !== 'production') {
+    console.warn(
+      `This utility of parsing text received "${fontAsString}" as an argument but takes only 1 argument and should have no spaces`,
```

Comment:
Perhaps: `Invalid single measurement value: "${fontAsString}"`?

Reply:
Lot better, I updated it. Many thanks.